cli.js

```diff
@@ -1,124 +1,146 @@
-import { existsSync } from "./deps/fs.js";
-import { parse } from "./deps/flags.js";
-import { brightGreen } from "./deps/colors.js";
-import { join } from "./deps/path.js";
-import lume from "./mod.js";
+import {parse} from "./deps/flags.js";
+import {bold, brightGreen, red} from "./deps/colors.js";
 import upgrade from "./cli/upgrade.js";
 import update from "./cli/update.js";
 import init from "./cli/init.js";
-import help from "./cli/help.js";
 import build from "./cli/build.js";
+import run from "./cli/run.js";
+
+export const version = "v0.14.0";
+
+const USAGE = `
+    Docs: https://lumeland.github.io/
+
+        To build the site:
+            lume
+
+        To serve the site in localhost
+            lume --serve
+
+        To update lume to the latest version
+            lume update
+
+        To run a custom script
+            lume run script-name
+
+        To get help with a command
+            lume COMMAND --help
+
+    USAGE:
+        lume [COMMAND] [OPTIONS]
+
+    COMMANDS:
+        build      Build the site. It's the default command
+        init       Creates a _config.js file
+        run        Run an site script
+        upgrade    Upgrade 🔥lume to the latest version
+        update     Update the version of the lume modules imported in a _config.js file
+
+
+    OPTIONS:
+        -h, --help              print usage information
+        -v, --version  Prints version information
+            --root <dir>        the root that lume should work in   Default: ./
+            --src  <dir>        the source directory for your site  Default: ./
+            --dest  <dir>       the build destination.              Default: _site
+            --config <file>     specify the lume config file.       Default: _config.js
+            --location <domain> set the domain for your site.       Default: http://localhost
+            --dev               enable dev mode (view draft pages)
+            
+            --serve             start a live-reloading web server
+            --port <port>       the port the server is on           Default: 3000
+`;
+
 
 if (import.meta.main) {
-  cli(Deno.args);
+  await cli(Deno.args);
```

Comment:
Is this `await` needed?

Reply:
I did have this in a `try...catch` at one point, to log error messages better. Not sure why I removed the `try...catch` 🤔 

WDYT? worth catching all errors and formatting a prettier error message, or just leave it and let the normal Deno error logging show?

Reply:
I have an utility function to show errors (https://github.com/lumeland/lume/blob/master/utils.js#L72, with much room for improvement).
A prettier error message whould be fine, errors in Deno aren't very friendly.